src/sparkx/Oscar.py

```diff
@@ -862,6 +864,60 @@ def lower_event_energy_cut(self,minimum_event_energy):
 
         return self
 
+    def space_cut(self, dim, cut_value_tuple):
```

Comment:
Maybe one could directly implement a function called `spacetime_cut`, which can also cut in the `t` dimension?